src/version/107/frontend/nl2go_manager.php

```diff
@@ -472,9 +472,9 @@ public function setUnsubscribe()
 			self::sendError('EMAILNOTPROVIDED');
 		}
 
-		$GLOBALS["DB"]->executeQuery('DELETE FROM ' . self::NEWSLETTER_SUBSCRIBER_TABLE . " WHERE cEmail = '$email'", 1);
+        $affectedRows = $GLOBALS["DB"]->executeQuery('DELETE FROM ' . self::NEWSLETTER_SUBSCRIBER_TABLE . " WHERE cEmail = '$email'", 3);
```

Comment:
what is the meaning of this 3?

Reply:
It determines the return type in this case the number of affected rows will be returned.